runtime/ops/web_worker.rs

```diff
@@ -16,26 +17,49 @@ pub fn init() -> Extension {
         op_sync(move |state, _args: (), buf: Option<ZeroCopyBuf>| {
           let buf = buf.ok_or_else(null_opbuf)?;
           let msg_buf: Box<[u8]> = (*buf).into();
```

Comment:
@inteon could you please rewrite those op handlers to be functions in this module instead of closures? It's becoming hard to read them